src/clplatea.jl

```diff
@@ -25,9 +25,14 @@
 export clplatea
 
 "The clamped plate problem (Strang, Nocedal, Dax)."
-function clplatea(p::Int=71, wght::Float64=-0.1)
+function clplatea(n::Int=5041, wght::Float64=-0.1)
 
-  nlp = Model()
+    p = floor(Int, sqrt(n))
+    p*p != n && warn("clplatea: number of variables adjusted from $n down to $(p*p) be square")
```

Comment:
Let's remove those "be square". I hadn't noticed them before and I can't tell what they mean.

Reply:
Could you please make those changes?